examples/js/loaders/SVGLoader.js

```diff
@@ -641,13 +665,13 @@ THREE.SVGLoader.prototype = Object.assign( Object.create( THREE.Loader.prototype
 			rx = Math.abs( rx );
 			ry = Math.abs( ry );
 
-			// Compute (x1′, y1′)
+			// Compute (x1â², y1â²)
```

Comment:
Um, this symbols look strange? 🤔 

BTW: Please run `node utils/modularize.js` once in order to update the jsm module.

Reply:
I think they were accent chars before. I will change them to apostrophes. Must have been my editor, I saw a warning in git about characters.

Reply:
> BTW: Please run `node utils/modularize.js` once in order to update the jsm module.

Should I commit the jsm? I've seen in the contribution guidelines that I shouldn't.

Reply:
>  I've seen in the contribution guidelines that I shouldn't.

Maybe there is a misunderstanding. You should always commit both files. However the workflow is editing the `js` file and generate the `jsm`.

Reply:
Understood. I've commited it.